README.md

```diff
@@ -192,6 +192,7 @@ Properties configurable at runtime:
 - `FLASKS3_ACTIVE`: whether to fetch static assets from s3/Cloudfront rather than the usual `url_for`.
 - `FLASKS3_CDN_DOMAIN`: CDN domain to fetch assets from if `FLASKS3_ACTIVE` is set to `True`
 - `FLASKS3_BUCKET_NAME`: S3 bucket assets are uploaded to and served to Cloudfront from.
+- `PERF_TEST`: Enable to allow access tokens generated via API to be accepted for performance testing.
```

Comment:
could this env be added to the .env template?

Reply:
Yes good point, will add it.